src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4923,6 +4923,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="TypeMustBeVar" xml:space="preserve">
     <value>The type must be 'var'.</value>
   </data>
+  <data name="WRN_TupleLiteralNameMismatch" xml:space="preserve">
+    <value>The tuple literal should probably not use element name '{0}' at this position.</value>
```

Comment:
I'd just say   "The tuple element name '{0}' is ignored because a different name is specified by the assignment target."